### PR TITLE
Removed HML and replaced with arbitrary floats

### PR DESCRIPTION
**Pull request by anshulm07**
Removed HML and replaced all uses of HML in tactics with a float that will be changed later

### CODE REVIEW

**Review by HussainGynai — CHANGES_REQUESTED** [review]
Change and test robocup-software/rj_gameplay/tests/stp/role/test_naive_assignment.py and test_cost.py

**Review by kasohrab — CHANGES_REQUESTED** [review]
resolve merge conflicts

**Review by kasohrab — CHANGES_REQUESTED** [review]
could you run make pretty-lines to fix the style errors?

File: rj_gameplay/stp/role/assignment/naive.py [hunk 1]
```diff
@@ -32,12 +32,16 @@ def get_sorted_requests(requests: assignment.FlatRoleRequests) -> SortedRequests
...
         request: role.RoleRequest
 
-        sorted_requests: SortedRequests = [
-            {} for _ in range(role.Priority.NUM_PRIORITIES)
-        ]
-
+        request_dict = {}
         for role_id, request in requests.items():
-            sorted_requests[int(request.priority)][role_id] = request
+            key = int(request.priority)
```
Comment (HussainGynai) [inline rj_gameplay/stp/role/assignment/naive.py:37]: Why change priority from float to int? Here 1.1 and 1.2 would be given the same priority, when we don't want that.

File: rj_gameplay/stp/role/assignment/naive.py [hunk 2]
```diff
@@ -85,9 +89,9 @@ def compute_costs_matrix(
                     continue
 
                 # TODO(1715): Make this cost infinite
-                if not robot.visible:
-                    robot_costs[robot_idx, request_idx] = 1e9
-                    continue
+                # if not robot.visible:
+                #     robot_costs[robot_idx, request_idx] = 1e9
+                #     continue
```
Comment (HussainGynai) [inline rj_gameplay/stp/role/assignment/naive.py:94]: Delete these commented lines

File: rj_gameplay/tests/stp/role/test_naive_assignment.py [hunk 3]
```diff
@@ -323,10 +331,17 @@ def test_compute_costs_matrix() -> None:
     cost_fn_c = TestCostFn(cost_c)
 
     # Create the requests of same priority.
+    # TODO change priority float to something useful
     requests: FlatRoleRequests = {
+<<<<<<< HEAD
```
Comment (HussainGynai) [inline rj_gameplay/tests/stp/role/test_naive_assignment.py:336]: This file has a lot merge conflicts, pls fix.

**Review by HussainGynai — CHANGES_REQUESTED** [review]
Fix merge errors, and don't cast to int for priority, just leave it as a float.

**Comment by kfu02** [comment]
superseded by https://github.com/RoboJackets/robocup-software/pull/1811

**Pull request closed without merging**